sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/FilteredAbstractCheck.kt

```diff
@@ -0,0 +1,16 @@
+package org.sonarsource.kotlin.api
+
+import org.jetbrains.kotlin.psi.KtCallExpression
+import org.jetbrains.kotlin.resolve.calls.callUtil.getResolvedCall
+import org.jetbrains.kotlin.resolve.calls.model.ResolvedCall
+import org.sonarsource.kotlin.plugin.KotlinFileContext
+
+abstract class FilteredAbstractCheck : AbstractCheck() {
+    abstract fun functionsToVisit(): List<FunMatcher>
+    abstract fun visitCallFiltered(resolvedCall: ResolvedCall<*>, callExpression: KtCallExpression, kotlinFileContext: KotlinFileContext)
```

Comment:
A minor thing, but I'd rather put `callExpression` as a first argument.

How about `visitFunctionCall` name for this method?